core/services/ocr2/plugins/median/services.go

```diff
@@ -226,7 +226,7 @@ func (m *medianContract) LatestTransmissionDetails(ctx context.Context) (configD
 func (m *medianContract) LatestRoundRequested(ctx context.Context, lookback time.Duration) (configDigest ocr2types.ConfigDigest, epoch uint32, round uint8, err error) {
 	var resp latestRoundRequested
 
-	err = m.chainReader.GetLatestValue(ctx, m.contract, "LatestRoundReported", map[string]string{}, &resp)
+	err = m.chainReader.GetLatestValue(ctx, m.contract.Name, "LatestRoundReported", map[string]string{}, &resp)
```

Comment:
Pre-existing, but while I'm thinking of it can we change this to `LatestRoundRequested`?

Reply:
Can do, I don't see the contract name being set anywhere.  I'll change it to "median" since that's what we've used in the evm setup?